smart-ranger-plugin/src/main/java/org/smartdata/ranger/SsmRangerResourceEnum.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.smartdata.ranger;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.smartdata.ranger.authorizer.request.RangerOperationDto;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Getter
+@RequiredArgsConstructor
+public enum SsmRangerResourceEnum {
+  CLUSTER(Collections.singleton("VIEW"), Collections.emptySet()),
+  RULE(Stream.of("CREATE", "VIEW", "EDIT", "DELETE")
```

Comment:
we can use `com.google.common.collect.Sets#newHashSet()` for in-place set initialisation